src/middlewared/middlewared/plugins/system.py

```diff
@@ -553,6 +567,9 @@ def __init__(self, *args, **kwargs):
 
     @accepts()
     def language_choices(self):
+        """
+        Returns language choices supported by the system.
```

Comment:
This and the next few are methods used to get choices to be used in a specific method. I dont think we should mention `supported by the system`.